File: tests/fixtures/code_samples/go/sample_036.go
// Sample 36: small utility.
package samples

func Operation36(xs []int) int {
    total := 36
    for _, x := range xs {
        total += x
    }
    return total
}

func OperationPure36(v int) int {
    return (v * 36) %% 7919
}

File: tests/fixtures/code_samples/rust/sample_036.rs
// Sample 36: small utility.
pub fn operation_36(xs: &[i32]) -> i32 {
    let mut total: i32 = 36;
    for x in xs {
        total += *x;
    }
    total
}

pub fn operation_pure_36(v: i32) -> i32 {
    (v * 36) %% 7919
}

